src/components/layout/header/view.tsx

```diff
@@ -7,116 +7,97 @@ import {
+
+  const { t } = useTranslation("template");
   return (
-    <Menu>
-      <MenuButton>
+    <MenuRoot>
+      <MenuTrigger>
         <If condition={isLogged}>
           <Then>
-            <Avatar name={user?.userInfo?.username ?? t('menu.user')} src="avatarSource" />
+            <Avatar.Root>
+              <Avatar.Fallback name={user?.userInfo?.username ?? t("menu.user")} />
+            </Avatar.Root>
           </Then>
           <Else>
-            <Avatar icon={<MdQuestionMark />} />
+            <Avatar.Root>
+              <Avatar.Icon />
+            </Avatar.Root>
           </Else>
         </If>
-      </MenuButton>
-      <MenuList>
-        <MenuGroup title={t('menu.options')}>
-          <MenuItem
-            as={Switch}
-            onChange={toggleColorModeWithDelay}
-            isChecked={isChecked}
-          >
-            {t('menu.dark')}
+      </MenuTrigger>
+      <MenuContent>
+        <MenuItemGroup title={t("menu.options")}>
+          <MenuItem value={"mode-switch"}>
+            Theme <ColorModeButton />
           </MenuItem>
-        </MenuGroup>
-        <MenuDivider />
-        <MenuGroup title={t('menu.account')}>
+        </MenuItemGroup>
+        <MenuSeparator />
+        <MenuItemGroup title={t("menu.account")}>
           <If condition={isLogged}>
             <Then>
               <WrapItem>
-                <MenuItem>{user?.userInfo?.username ?? t('menu.user')}</MenuItem>
+                <MenuItem value={user?.userInfo?.username ?? t("menu.user")}>
+                  {user?.userInfo?.username ?? t("menu.user")}
+                </MenuItem>
               </WrapItem>
-              <MenuItem onClick={async () => dispatch(Logout())}>{t('menu.exit')}</MenuItem>
+              <MenuItem value={"exit"} onClick={async () => dispatch(Logout())}>
+                {t("menu.exit")}
+              </MenuItem>
             </Then>
             <Else>
               <WrapItem>
-                <MenuItem onClick={async () => dispatch(Login())}>{t('menu.login')}</MenuItem>
+                <MenuItem value={"login"} onClick={async () => dispatch(Login())}>
+                  {t("menu.login")}
+                </MenuItem>
               </WrapItem>
             </Else>
           </If>
-        </MenuGroup>
-      </MenuList>
-    </Menu>
-  )
-}
+        </MenuItemGroup>
+      </MenuContent>
+    </MenuRoot>
+  );
+};
 
 const Header = () => {
   const { isMobileSiderOpen, setMobileSiderOpen } = useLayoutContext();
   return (
-    <Box
-      as="header"
-      shadow={'lg'}
-      bg={'header.bg'}
-    >
-      <Flex
-        paddingTop={'5px'}
-        paddingBottom={'5px'}
-      >
-        <HStack spacing={2} pl={2}>
+    <Box as="header" shadow={"sm"} bg={"header"}>
+      <Flex paddingTop={"5px"} paddingBottom={"5px"}>
+        <HStack gap={2} pl={2}>
           <LinkBox>
             <Center>
-              <LinkOverlay as={Link} to={"/"}>
+              <LinkOverlay asChild href={"/"}>
```

Comment:
this is not right as is needed to use the Link component from the Router with the to. 
this is an hardcoded <a> not using the Link from Router.